docs/Users_Guide/glossary.rst

```diff
@@ -13319,6 +13269,772 @@ METplus Configuration Glossary
 
      | *Used by:* WaveletStat
 
+   PAIR_STAT_PAIRS_INPUT_DIR
+     Input directory for pairs files to use with the MET tool pair_stat.
+
+     | *Used by:*  PairStat
+
+   PAIR_STAT_PAIRS_INPUT_TEMPLATE
+     Template used to specify pairs input filenames for the MET tool pair_stat.
+
+     | *Used by:*  PairStat
+
+   PAIR_STAT_OUTPUT_DIR
+     Specify the directory where output files from the MET pair_stat tool are written.
+
+     | *Used by:*  PairStat
+
+   PAIR_STAT_OUTPUT_TEMPLATE
```

Comment:
Just want to confirm that this is used to set the `-out filename` command line option and not the `-oudir directory` command line option that is no longer used.